### ISSUE TITLE
ElecLab 7.4 Inch 1280x400 resolution issue

### ISSUE THREAD

**Issue by Vomega10** [eleclab-rpi/Display_Firmware#2]
I can't seem to get the monitor to display the correct resolution of 1280*400.  For some reason when I plug it directly into the HDMI source on an X86 computer, it defaults to 1920*480 and gives me no options to change it. I have tried several different computers and docks. The only way I was able to achieve the correct 1280*400 resolution, was by plugging it into an external EGPU in a Razor Core X. 

Do I need to update the firmware or somehow change the resolution in the blocks?

**Comment by eleclab-rpi** [comment]
Please refresh its firmware, just use a USB Type-c cable and tweezers, please refer to here: https://github.com/eleclab-rpi/Display_Firmware
In addition, this type of long screen is a cutting screen, so it requires the use of a fixed 1280x480 timing, and the EDID information in HDMI has been modified.



***@***.***
 
From: Vomega10
Date: 2024-01-10 05:47
To: eleclab-rpi/Display_Firmware
CC: Subscribed
Subject: [eleclab-rpi/Display_Firmware] ElecLab 7.4 Inch 1280x400 resolution issue (Issue #2)
I can't seem to get the monitor to display the correct resolution of 1280400. For some reason when I plug it directly into the HDMI source on an X86 computer, it defaults to 1920480 and gives me no options to change it. I have tried several different computers and docks. The only way I was able to achieve the correct 1280*400 resolution, was by plugging it into an external EGPU in a Razor Core X.
Do I need to update the firmware or somehow change the resolution in the blocks?

**Comment by eleclab-rpi** [comment]
Sorry, it's 1280x400 resolution.



***@***.***
 
From: Vomega10
Date: 2024-01-10 05:47
To: eleclab-rpi/Display_Firmware
CC: Subscribed
Subject: [eleclab-rpi/Display_Firmware] ElecLab 7.4 Inch 1280x400 resolution issue (Issue #2)
I can't seem to get the monitor to display the correct resolution of 1280400. For some reason when I plug it directly into the HDMI source on an X86 computer, it defaults to 1920480 and gives me no options to change it. I have tried several different computers and docks. The only way I was able to achieve the correct 1280*400 resolution, was by plugging it into an external EGPU in a Razor Core X.
Do I need to update the firmware or somehow change the resolution in the blocks?

**Comment by Vomega10** [comment]
I can't seem to put the display in DFU mode. I'm following the instructions. I'm kindof bummed it was not a plug and play device.

**Comment by eleclab-rpi** [comment]
It is a USB device, but only shorting the startup pin with tweezers will turn it into DFU mode, so just short the tweezers well before powering on.



***@***.***
 
From: Vomega10
Date: 2024-01-11 08:51
To: eleclab-rpi/Display_Firmware
CC: ElecLab; Comment
Subject: Re: [eleclab-rpi/Display_Firmware] ElecLab 7.4 Inch 1280x400 resolution issue (Issue #2)
I can't seem to put the display in DFU mode. I'm following the instructions. I'm kindof bummed it was not a plug and play device.

**Comment by Vomega10** [comment]
Do I leave it powered on when I do that? Do I leave the hdmi and usb cable plugged in? the display doesn't have an on off switch. How do I know if I have shorted the well? I feel like I did what the instructions depicted and didn't get into DFU mode.

**Comment by eleclab-rpi** [comment]
Just follow these simple steps:
1. Disconnect the USB cable and HDMI cable on the display side
2. Use tweezers to short-circuit the boot pin as shown in the picture in the PDF document.
3. Plug in the USB cable and connect the display and computer
4. When you hear the computer prompt that a device is plugged in, you can disconnect the tweezers.
5. Follow the prompts in the PDF document to continue....



***@***.***
 
From: Vomega10
Date: 2024-01-11 10:41
To: eleclab-rpi/Display_Firmware
CC: ElecLab; Comment
Subject: Re: [eleclab-rpi/Display_Firmware] ElecLab 7.4 Inch 1280x400 resolution issue (Issue #2)
Do I leave it powered on when I do that? Do I leave the hdmi and usb cable plugged in? the display doesn't have an on off switch. How do I know if I have shorted the well? I feel like I did what the instructions depicted and didn't get into DFU mode.

**Comment by eleclab-rpi** [comment]
Of course, make sure the USB Touch switch is pulled to the PC position. This way the display can transmit the USB signal to the USB Type-c socket instead of the Raspberry Pi's 2x2 2mm black header.



***@***.***
 
From: Vomega10
Date: 2024-01-11 10:41
To: eleclab-rpi/Display_Firmware
CC: ElecLab; Comment
Subject: Re: [eleclab-rpi/Display_Firmware] ElecLab 7.4 Inch 1280x400 resolution issue (Issue #2)
Do I leave it powered on when I do that? Do I leave the hdmi and usb cable plugged in? the display doesn't have an on off switch. How do I know if I have shorted the well? I feel like I did what the instructions depicted and didn't get into DFU mode.

**Comment by Vomega10** [comment]
yeah, still not working. You need 3 hands to short the well and plug in a cable at the same time. And how would I even know if it was shorted? I'm really disappointed in this "plug n play" display. Not user friendly at all.

**Comment by eleclab-rpi** [comment]
Sorry, the firmware upgrade function of the display screen is generally an in-depth DIY function, so it is not designed to be very easy to operate like ordinary DIY products. After all, this is a rarely required function that requires proficiency and is generally used for internal production in factories.



***@***.***
 
From: Vomega10
Date: 2024-01-16 02:07
To: eleclab-rpi/Display_Firmware
CC: ElecLab; Comment
Subject: Re: [eleclab-rpi/Display_Firmware] ElecLab 7.4 Inch 1280x400 resolution issue (Issue #2)
yeah, still not working. You need 3 hands to short the well and plug in a cable at the same time. And how would I even know if it was shorted? I'm really disappointed in this "plug n play" display. Not user friendly at all.
